src/components/ZclEndpointCard.vue

```diff
@@ -408,6 +408,10 @@ export default {
             this.selectedReporting.push(resolvedReference)
         })
       })
+    },
+    handleEndpointCardClick(endpointReference) {
```

Comment:
When do we put a function on common-mixing vs here? If more than one places calls this function?
Add comments to this method?